pub-kube-config-files/composite-orgs-transformer.yaml

```diff
@@ -1,3 +1,38 @@
+kind: PersistentVolume
```

Comment:
Do the transformers really need an EBS volume? Actually what I mean is do they write something that they need the next time the transformer is restarted?